### PR TITLE
Tempus locus

### PR DESCRIPTION
**Pull request by tabreturn**
# New Port for Tempus locus

## Game Information
- **Title**: Tempus locus
- **URL**: https://nubitoad.itch.io/tempus-locus

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/tempuslocus/Tempus Locus.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+source $controlfolder/control.txt
+# device_info.txt will be included by default
+
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+export PORT_32BIT="Y"
+GAMEDIR="/$directory/ports/tempuslocus"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
```
Comment (Cebion) [inline ports/tempuslocus/Tempus Locus.sh:24]:
```suggestion
export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$LD_LIBRARY_PATH"
```

File: ports/tempuslocus/Tempus Locus.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+cd $GAMEDIR
+
+# Check for file existence before trying to manipulate them:
+[ -f "./gamedata/data.win" ] && mv gamedata/data.win gamedata/game.droid
+[ -f "./gamedata/game.win" ] && mv gamedata/game.win gamedata/game.droid
+[ -f "./gamedata/game.unx" ] && mv gamedata/game.unx gamedata/game.droid
+
+# Make sure uinput is accessible so we can make use of the gptokeyb controls
+$ESUDO chmod 666 /dev/uinput
```
Comment (Cebion) [inline ports/tempuslocus/Tempus Locus.sh:40]:
```suggestion
```

File: ports/tempuslocus/Tempus Locus.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+cd $GAMEDIR
+
+# Check for file existence before trying to manipulate them:
+[ -f "./gamedata/data.win" ] && mv gamedata/data.win gamedata/game.droid
+[ -f "./gamedata/game.win" ] && mv gamedata/game.win gamedata/game.droid
+[ -f "./gamedata/game.unx" ] && mv gamedata/game.unx gamedata/game.droid
+
+# Make sure uinput is accessible so we can make use of the gptokeyb controls
```
Comment (Cebion) [inline ports/tempuslocus/Tempus Locus.sh:39]:
```suggestion
```

File: ports/tempuslocus/Tempus Locus.sh [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+$ESUDO chmod 666 /dev/uinput
+
+$GPTOKEYB "gmloader" -c ./tempuslocus.gptk &
+
+$ESUDO chmod +x "$GAMEDIR/gmloader"
+
```
Comment (Cebion) [inline ports/tempuslocus/Tempus Locus.sh:45]:
```suggestion
pm_platform_helper $GAMEDIR/gmloader
```

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by cdeletre — DISMISSED** [review]
`game.apk` is missing

nvm I saw it when posting....

**Review by cdeletre — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**